### PR TITLE
change reset-password design to current design

### PR DESCRIPTION
**Pull request by sammiearchie77**

### CODE REVIEW

File: control/src/pages/pricing/pricingPageSection1/PricingPageSection1.module.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+html,
```
Comment (ickynavigator) [inline control/src/pages/pricing/pricingPageSection1/PricingPageSection1.module.css:1]: no non class selectors

File: package.json [hunk 2]
```diff
@@ -5,19 +5,18 @@
   "license": "MIT",
   "scripts": {
     "start": "serve -s dist/ -p 9001",
-    "open": "opener http://localhost:9000",
     "dev:root": "webpack serve --port 9000 --env isLocal",
     "dev:topbar": "npm run dev --prefix topbar",
     "dev:control": "npm run dev --prefix control",
     "dev:sidebar": "npm run dev --prefix sidebar",
-    "dev": "npm-run-all --parallel \"dev:*\" \"open\"",
+    "dev": "npm-run-all --parallel dev:*",
```
Comment (ickynavigator) [inline package.json:12]: readd the open script

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Comment by ickynavigator** [comment]
resolve conflicts

File: package.json (hunk 2 above)
Comment (sammiearchie77) [reply package.json:12]: okay

**Pull request closed without merging**